src/uu/ls/src/ls.rs

```diff
@@ -1504,6 +1566,7 @@ fn display_item_long(
     item: &PathData,
     max_links: usize,
     max_size: usize,
+    _max_blk_width: usize,
```

Comment:
Variables prefixed with an underscore are usually unused. It's better to add a `cfg` on the parameter and drop the underscore.